sensibo/StateHandler.js

```diff
@@ -79,7 +79,9 @@ module.exports = (device, platform) => {
 			if (prop === 'smartMode') {
 				try {
 					log.easyDebug(`${device.name} - Setting Climate React state to ${value}`)
-					sensiboApi.enableDisableClimateReact(device.id, value)
+					const sensiboNewClimateReactState = unified.sensiboFormattedClimateReactState(device, state)
+
```

Comment:
Any and all changes. Basically, instead of having a separate code path for updating the single "enabled" value for the climate react enabled switch we simply push the internal state to Sensibo's API so they are always in sync. This could be a change in the "enabled" property, or any other property (or properties, plural).